docs/src/modeler_guide/data.md

```diff
@@ -1,6 +1,7 @@
-# Saving and Viewing PowerSystems Data
+# Viewing PowerSystems Data in the Terminal
```

Comment:
While clicking through the documentation, this page seems out of place. Do we need a `jq` guide here? I recommend moving it later into the documentation. 

Also, I think seeing the `jq` output here could be valuable to show, to motivate why `jq` is useful / showcase some of the structure of the `json` data. Right now it feels like a homework assignment: I need to open a `Julia` REPL, import a system, convert to a `json`, install `jq`, and run the commands in the documentation to get a sense of why this is useful.

Reply:
@daniel-thom do you think we should eliminate this? or put it in the model developer section?

Reply:
I will defer to you guys, @kdheepak @claytonpbarrows @jd-lara.  I have found these instructions incredibly helpful in some situations (though not common).

The current instructions do not match the current serialization format.  The examples below are corrected and illustrate the benefit.

Suppose you have a system with 5 ThermalStandard generators that has been serialized to `sys.json`.

1. Show the names.
```
jq '.data.components | .[] | select(.__metadata__.type == "ThermalStandard") | .name' system.json
"322_CT_6"
"321_CC_1"
"202_STEAM_3"
"315_STEAM_1"
"223_CT_4"
```

2. Show one generator.
```
jq '.data.components | .[] | select(.__metadata__.type == "ThermalStandard") | select(.name == "322_CT_6") ' system.json
{
  "base_power": 100,
  "rating": 0.5818934610390463,
  "services": [
    {
      "value": "9c92134d-e952-4ce7-9fd4-5265269a4633"
    },
    {
      "value": "0e734812-31f1-4850-abe7-2687b2afc14e"
    },
    {
      "value": "5193adcc-8541-413f-b70a-774b032758dd"
    },
    {
      "value": "f543eb98-e9b1-46b4-94db-7827afefd1c7"
    },
    {
      "value": "2b5feecc-203a-4752-98f2-0862d3452f05"
    }
  ],
  "bus": {
    "value": "0746bf39-8308-4638-a0e7-0cff83d29f05"
  },
  "available": true,
  "time_series_container": [],
  "name": "322_CT_6",
  "status": true,
  "active_power": 0.55,
...
}
```

My thoughts:
1. This syntax is certainly difficult for the unfamiliar.  But if all you have to do is a copy a command, it's not that hard.
2. The output shows the innards of PowerSystems, for better or worse.
3. It isn't for everyone.  However, if you have multiple systems with large numbers of components that you created six months ago, this can help you figure out what you want to load.
4. If it stays it should be in the main user section. It's not specific to a model developer.
5. If no one else cares about this kinda thing then I can just keep a local copy.

Reply:
I think we should leave it in for now. I think it's in the appropriate section, just so long as it's at the end of the section. If we learn that it causes confusion, we can address later.